src/components/MultiHopTrade/components/TradeInput/components/MaxSlippage.tsx

```diff
@@ -0,0 +1,180 @@
+import { Divider, Flex, Skeleton, Stack } from '@chakra-ui/react'
+import { Tag } from '@chakra-ui/tag'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import type { AmountDisplayMeta, SwapSource } from '@shapeshiftoss/swapper'
+import { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { Row } from 'components/Row/Row'
+import { Text } from 'components/Text'
+import type { TextPropTypes } from 'components/Text/Text'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { useLocaleFormatter } from 'hooks/useLocaleFormatter/useLocaleFormatter'
+import { fromBaseUnit } from 'lib/math'
+import {
+  THORCHAIN_LONGTAIL_STREAMING_SWAP_SOURCE,
+  THORCHAIN_STREAM_SWAP_SOURCE,
+} from 'lib/swapper/swappers/ThorchainSwapper/constants'
+import { selectUserSlippagePercentage } from 'state/slices/tradeInputSlice/selectors'
+import { selectDefaultSlippagePercentage } from 'state/slices/tradeQuoteSlice/selectors'
+import {
+  convertDecimalPercentageToBasisPoints,
+  subtractBasisPointAmount,
+} from 'state/slices/tradeQuoteSlice/utils'
+import { useAppSelector } from 'state/store'
+
+type MaxSlippageProps = {
+  swapSource?: SwapSource
+  isLoading?: boolean
+  amountCryptoPrecision: string
+  slippageDecimalPercentage: string
+  hasIntermediaryTransactionOutputs?: boolean
+  intermediaryTransactionOutputs?: AmountDisplayMeta[]
+  symbol: string
+}
+
+export const MaxSlippage: React.FC<MaxSlippageProps> = ({
+  swapSource,
+  isLoading,
+  amountCryptoPrecision,
+  slippageDecimalPercentage,
+  hasIntermediaryTransactionOutputs,
+  intermediaryTransactionOutputs,
+  symbol,
+}) => {
+  const translate = useTranslate()
+  const {
+    number: { toCrypto },
+  } = useLocaleFormatter()
+  const shouldShowSlippageMinAmount = useMemo(
```

Comment:
Ditto if we use a common util, then this can simply be `!isStreamingSwapSource`

Reply:
likewise here, can do in follow-up since pr is uuuge :)